analysis/table1/table1.R

```diff
@@ -31,11 +31,18 @@ print(length(args))
 if (length(args) == 0) {
   cohort <- "vax"
   age_str <- "18;40;65;85;111"
+  preex <- FALSE # "None", TRUE, or FALSE
```

Comment:
I think this should be:
```suggestion
  preex <- FALSE # "All", TRUE, or FALSE
```

Reply:
Good spot, I played around a little with naming and hadn't updated the comment